### DISCUSSION TITLE
Editable Table

### DISCUSSION THREAD

**Discussion by shashank-452** [vazco/uniforms#1267] [Q&A] [answered]
Hi Team,

We are working on incorporating uniforms for our application. There are a number of pages/screens in our applications where we have a need for an Editable Table. 
So we wanted to know if its already on the roadmap or we would need to do a contribution or customization from our end.

**Comment by radekmie** [comment, marked as answer]
Hi @shashank-452. There are no plans to provide any custom form components as a part of the uniforms project. However, if you'd build one or need help with that - don't hesitate to share here.
